Tests/LibSQL/TestSqlDatabase.cpp

```diff
@@ -20,29 +20,35 @@ NonnullRefPtr<SQL::TableDef> setup_table(SQL::Database&);
 void insert_into_table(SQL::Database&, int);
 void verify_table_contents(SQL::Database&, int);
 void insert_and_verify(int);
+void commit(SQL::Database&);
 
 NonnullRefPtr<SQL::SchemaDef> setup_schema(SQL::Database& db)
 {
     auto schema = SQL::SchemaDef::construct("TestSchema");
-    db.add_schema(schema);
+    auto maybe_error = db.add_schema(schema);
+    EXPECT(!maybe_error.is_error());
     return schema;
 }
 
 NonnullRefPtr<SQL::TableDef> setup_table(SQL::Database& db)
 {
     auto schema = setup_schema(db);
     auto table = SQL::TableDef::construct(schema, "TestTable");
-    db.add_table(table);
     table->append_column("TextColumn", SQL::SQLType::Text);
     table->append_column("IntColumn", SQL::SQLType::Integer);
     EXPECT_EQ(table->num_columns(), 2u);
-    db.add_table(table);
+    auto maybe_error = db.add_table(table);
+    EXPECT(!maybe_error.is_error());
+    if (maybe_error.is_error())
+        outln("{}", maybe_error.error());
```

Comment:
Since `EXPECT` will stop the test's execution, the `outln()` will never be called.
Should this be the other way around, maybe? Or is it maybe debug code?

Reply:
I am reasonably sure that a failing `EXPECT` won't stop the execution of the test, it will continue as normal.

Reply:
Huh. Good point, I thought it stopped execution but it seems like it doesn't.
In any case, since `maybe_error` shouldn't be an error, this seems like a debug print and should probably not be in the test's code.